typing/typecore.ml

```diff
@@ -4098,8 +4096,10 @@ and type_expect_
       with Exit ->
         let arg = match sarg with
         | None -> None
-        | Some sarg -> Some (type_exp env argument_mode sarg,
-                        register_allocation expected_mode)
+        | Some sarg ->
+            let arg = type_exp env argument_mode sarg in
+            let alloc_mode = register_allocation expected_mode in
+            Some ( arg, alloc_mode )
```

Comment:
Sorry to keep on about style things, but I really dislike spaces within parentheses and they are very unusual for OCaml code.

Reply:
no problem - I also fixed several others in `typecore`.